script2.js

```diff
@@ -0,0 +1,71 @@
+// 1. Programming Basics
+// 'Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'
+// Create an array of the dishes listed above. Loop through the array using aforloop.
+// 1.Log out all the dishes.
+// 2.Log out a randomly chosen dish (randomly generated  index)
+
+const dishes = ['Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup', 'Salmon'];
+
+for (let i = 0; i < dishes.length; i++) {
+    console.log(dishes[i]);
+}
+
+const rand = Math.floor(Math.random() * dishes.length);
+console.log(`Random: ${dishes[rand]}`);
+
+// 2. DOM manipulation
+// Using JavaScript, create a button and add it to the html (the document)
+// When the button is clicked:
+// -Insert an h1 tag below the button with the text “Javascripttest”.
+// -Change the background color of the document body
+
+const button = document.createElement("button");
+button.innerText = "Click!";
+document.body.append(button);
+
+button.addEventListener("click", () => {
+    const header = document.createElement("h1");
+    button.after(header);
+    header.innerText = "“Javascripttest”";
+
+    document.body.style.backgroundColor = (document.body.style.backgroundColor == '') ? 'green' : '';
+})
+
```

Comment:
Good 👍🏽 
A possible reconsideration would be to not create and populate the inner HTML of a new header on every click.
You could have it ready outside of the event listener callback and just use it when needed, as it is an element that does not change and therefore there is not need to load that part of the process onto the event callback.
Nice extra touches with `.after` and the tertiary operator 👍🏽